app/reducers/optional.js

```diff
@@ -1,7 +0,0 @@
-//this is a simple pass thru that offers an extension point
```

Comment:
Now if I add my own index reducer as:

````js
import foo from './foo';

export default {
  foo
};
````

And then add `reducers/foo.js` as:

````js
const reducer = (state = {}, action) => {
  return {};
}
````

it fails with the error:

![image](https://cloud.githubusercontent.com/assets/980757/18568106/50faecb2-7b93-11e6-8bf9-ddc0bab27151.png)